substrafl_examples/get_started/run_mnist_torch.py

```diff
@@ -62,7 +63,7 @@
 
 
 # Store organization IDs
-ORGS_ID = list(clients.keys())
+ORGS_ID = list(clients)
```

Comment:
I am concerned this will break things later on.

This might run in subprocess mode where permission are not enforced but should fail in remote mode later on when we specify:
`authorized_ids=[ALGO_ORG_ID]`

Reply:
I'm not sure I'm following, `clients` is a Python dictionary, so iterating on it yield the keys. 
`ORGS_ID` is just a list of string ids, unless I'm missing something.